### PR TITLE
Fixed #27029 -- Added validator for utf-8 compatible email adresses

### PR DESCRIPTION
**Pull request by woutdp**
Link to trac ticket:
https://code.djangoproject.com/ticket/27029

Worked further on some of the changes in https://github.com/django/django/pull/7039 of https://github.com/claudep

### CODE REVIEW

**Comment by woutdp** [comment]
Should I add `utf` to the spelling_wordlist file or should I change the name of the instance from `validate_email_utf8` to `validate_email_UTF8`. As I understand the build fails because `utf` is not a valid word. `UTF` is already in the approved spelling_wordlist file.

File: docs/ref/validators.txt [hunk 1]
```diff
@@ -174,6 +174,15 @@ to, or in lieu of custom ``field.clean()`` methods.
 
     An :class:`EmailValidator` instance without any customizations.
 
+``validate_email_utf8``
+------------------
```
Comment (felixxm) [inline docs/ref/validators.txt]: Title underline should be as long as title.

File: docs/ref/validators.txt [hunk 2]
```diff
@@ -174,6 +174,15 @@ to, or in lieu of custom ``field.clean()`` methods.
...
+
+.. data:: validate_email_utf8
+
+    An :class:`EmailValidator` instance with support for utf-8
```
Comment (felixxm) [inline docs/ref/validators.txt]: utf -> UTF

File: docs/releases/1.11.txt [hunk 3]
```diff
@@ -393,6 +393,9 @@ Validators
   :data:`~django.core.validators.validate_image_file_extension` to validate
   image files.
 
+* Added :data:`django.core.validators.validate_email_utf8` to validate email
+  addresses that support utf-8 in the local-part of the address.
```
Comment (felixxm) [inline docs/releases/1.11.txt]: utf -> UTF

**Comment by timgraham** [comment]
See Florian's [comment on the ticket](https://code.djangoproject.com/ticket/27029#comment:21):

> I am against this patch, adding more regular expressions is the wrong way to go. I'd like to propose to change the current email validator to just check if "@" is in the address and be done with it. See also ​https://davidcel.is/posts/stop-validating-email-addresses-with-regex/ -- I think this is something which should have a bit of discussion on the mailing list."

Closing for now.

**Pull request closed without merging**